src/pages/booking.js

```diff
@@ -0,0 +1,90 @@
+import styles from "@/styles/booking.module.scss";
+import { useState } from "react";
+
+export function getServerSideProps(context)
+{
+    return {
+        props: {
+            showtimeId: context.query.showtimeId,
+            showtimeSalong: context.query.showtimeSalong,
+            showtimedate: context.query.showtimeDate,
+            title: context.query.title,
+            poster: context.query.poster,
+            length: context.query.length
+        }
+    }
+}
+
+export default function Booking({ showtimeId, showtimeSalong, showtimedate, title, poster, length })
+{
+    const [adultPrice, setAdultPrice] = useState(150);
+    const [childPrice, setChildPrice] = useState(100);
```

Comment:
I'm wondering why you're using useState for these as there is nowhere in the code for changing the price - using the set-functions?

Reply:
I did it that way for future implementations so the price could be changed. Not relevant now, but I left them in anyway.